requirements/dev.in

```diff
@@ -1,5 +1,6 @@
 -r base.txt
 
 fabric3
+pre-commit
```

Comment:
As you please try it via omitting it from here, as its already part of d3a-interface?

Reply:
I'm not sure whether I fully understood your comment. 
If you're suggesting removing this dependency because it's already present in d3a-interface, I would discourage this. Each repository is an independent package. We shouldn't rely on a dependency that is installed in another package (e.g. d3a-interface) because, if that package removes the dependency, things will break. We should treat our repos as if they were from a 3rd party vendor.

Reply:
Good point though I removed all such as part of recent dependency updates :-)

But anyways could be taken care later via ticket: https://gridsingularity.atlassian.net/browse/D3ASIM-3679

Reply:
Could you please write a comment about it on that ticket? I would consider this as a bug, not a best practice.